src/server/main_service.cc

```diff
@@ -714,17 +749,9 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
   if (under_script) {
     DCHECK(dfly_cntx->transaction);
     if (cid->IsTransactional()) {
-      OpStatus status = CheckKeysDeclared(*dfly_cntx->conn_state.script_info, cid, args.subspan(1),
```

Comment:
what has changed here? is it intentional?

Reply:
I just moved it to the validation part, previously it was validated immediately before execution